spring-social-zotero/src/main/java/org/springframework/social/zotero/api/impl/GroupsTemplate.java

```diff
@@ -350,4 +350,20 @@ public void deleteItem(String groupId, String citationKey, Long citationVersion)
         }
 
     }
+    
+    @Override
+    public void deleteMultipleItems(String groupId, String citationKey, Long citationVersion) throws ZoteroConnectionException {
```

Comment:
this method should get a list of keys, not one string.

Reply:
additionally, even if it should accept just one string, the parameter name is bad.